targets/unity-v2/source/Entity/ScreenTimeTracker.cs

```diff
@@ -137,11 +146,14 @@ public void OnApplicationFocus(bool isFocused)
         /// </summary>
         public void Send()
         {
-            if (PlayFabClientAPI.IsClientLoggedIn())
+            if ((PlayFabClientAPI.IsClientLoggedIn()) && (isSending == false))
             {
                 int eventsInTheBatch = 0;
+                isSending = true;
+
                 WriteEventsRequest request = new WriteEventsRequest();
                 request.Events = new List<EventContents>();
+                request.FlushToPlayStream = true;
```

Comment:
https://api.playfab.com/documentation/entity/method/WriteEvents
FlushToPlayStream is not a valid request parameter for this event.
Please make sure you're actually compiling and executing this using the published assetPackage:
https://github.com/PlayFab/UnitySDK/blob/master/Packages/UnitySDK.unitypackage
FlushToPlayStream causes a compile error in the project.

Reply:
fixed